server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -360,51 +336,28 @@ private void doPost(Iterable<E> envelopes, StreamObserver<Ack> observer) {
     protected abstract void store(Iterable<T> messages);
 
     /**
-     * Initializes the filter chain upon the first invocation and returns the initialized instance
-     * for all next calls.
-     *
-     * <p>Adds the {@link DeadMessageFilter} and the {@link ValidatingFilter} to the chain, so that
-     * a chain always has the following format:
-     *
-     * <pre>
-     *     Chain head -> {@link ValidatingFilter} -> {@link DeadMessageFilter} -> custom filters from {@linkplain BusBuilder Builder} -> chain tail.
-     * </pre>
-     *
-     * <p>The head and the tail of the chain are created by the {@code Bus} itself. Those are
-     * typically empty. Override {@link #filterChainHead()} and {@link #filterChainHead()} to add
-     * some filters to the respective chain side.
+     * A callback for derived classes to modify the order of filters used by the bus.
+     *
+     * <p>Default implementation inserts:
+     * <ol>
+     *   <li>a filter which validates incoming messages
+     *   <li>a filter for unhandled messages
```

Comment:
Please add punctuation.

Reply:
Assuming items are numbered and are parts of the same sentence additional punctuation won't help much.